build/images/ovs/Dockerfile.windows

```diff
@@ -0,0 +1,47 @@
+# Copyright 2023 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG NANOSERVER_VERSION
+
+FROM mcr.microsoft.com/windows/servercore:${NANOSERVER_VERSION} as windows-ovs
+
+ARG WIN_OVS_VERSION
+
+# Switch to a privileged user
+USER ContainerAdministrator
+
+# Sets the shell to be PowerShell 
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+
+ENV WIN_OVS_VERSION ${WIN_OVS_VERSION}
+
+# Install VC for ovs
+RUN curl.exe -Lo C:\vc_redist.x64.exe https://aka.ms/vs/17/release/vc_redist.x64.exe; \
+    powershell -Command  "Start-Process -FilePath C:\vc_redist.x64.exe -ArgumentList ‘/install’,’/quiet’,’/norestart’ -Verb RunAs -Wait"; \
+    rm C:\vc_redist.x64.exe
+
+# Download and extract OVS
+RUN curl.exe -fLo C:\ovs-${env:WIN_OVS_VERSION}-antrea.0-win64.zip https://downloads.antrea.io/ovs/ovs-${env:WIN_OVS_VERSION}-antrea.0-win64.zip; \
+    powershell -Command Expand-Archive C:\ovs-${env:WIN_OVS_VERSION}-antrea.0-win64.zip -DestinationPath C:\  ; \
+    rm C:\ovs-${env:WIN_OVS_VERSION}-antrea.0-win64.zip
+
+# Download, extract OpenSSL and copy libraries to ovs
+RUN curl.exe -Lo C:\openssl-1.0.2u-x64_86-win64.zip https://indy.fulgan.com/SSL/openssl-1.0.2u-x64_86-win64.zip; \
+    powershell -Command Expand-Archive C:\openssl-1.0.2u-x64_86-win64.zip -DestinationPath C:\  ;  \
+    cp C:/libeay32.dll C:/openvswitch/usr/sbin/ ; \
+    cp C:/ssleay32.dll C:/openvswitch/usr/sbin/ ; \
+    mv C:/libeay32.dll C:/openvswitch/usr/bin/ ; \
+    mv C:/ssleay32.dll C:/openvswitch/usr/bin/ ; \
+    mkdir C:/openvswitch/var/run/openvswitch ; \
+    Remove-Item -Path 'openssl-1.0.2u-x64_86-win64.zip', 'HashInfo.txt', 'License.txt', 'openssl.exe', 'ReadMe.txt', 'OpenSSL License.txt'
```

Comment:
RUN cmd /S /C setx /m PATH "C:/openvswitch/usr/bin;%PATH%"  does not work,
However, we can start cmd shell, which can be done at the end like below
SHELL ["cmd", "/S", "/C"] and then set the path, 
so  ovs-vsctl/ovs-ofctl/ovs-dpctl  commands can work directly,
 rather than going to their absolute path.

Reply:
Maybe you could try with powershell command for setting environment variables, no need to switch to cmd shell.

Reply:
I think adding binary path to run ovs commands in antrea-ovs container in the runtime works only when running container with hostProcess as false.